src/components/AboutTeam.tsx

```diff
@@ -0,0 +1,213 @@
+import steffenCornwellImg from '../static/images/steffenCornwell Small.png';
+import trangNguyenImg from '../static/images/trangNguyen Small.png';
+import yingTangImg from '../static/images/yingTang Small.png';
+
+const AboutTeam = () => {
+
+  return (
+    <div>
+      <Helmet>
+        <meta name="description" content="Keep.id" />
+      </Helmet>
+      <div className="bg-our-team-cover-img viewport-height-100 negative-margin-top">
+        <div className="container">
+          <div className="viewport-height-100 custom-vertical-center">
+            <div>
+              <h1 className="hero-header pb-4 text-white">
+              We are Team Keep.
+              </h1>
+              <p className="hero-header-2 text-white">
+                <img
+                  className="svg-white mb-5"
+                  src={RectangleSVG}
+                  alt="rectangle"
+                />
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <h1 className = "hero-header-aboutus text-black pb-3 ">Our board</h1>
+      <div className="profile-sizing profile-body">
+        <div className="grid">
+          <div className = "wrapper">
+            <ProfileCard
+              imgUrl = {ConnorChongImg}
+              title = {"Connor Chong"}
+              description={"Founder | CEO"}
+              info= {"#connor"}
+              id={"connor"}
+              infoadded={"my name is connor"}
+              />
+              <ProfileCard
+              imgUrl = {steffenCornwellImg }
+              title = {"Steffen Cornwell "}
+              description={"Nonprofit Liaison"}
+              info = {"#steffen"}
+              id = {"steffen"}
+              infoadded={""}
+              />
+              <ProfileCard
+              imgUrl = {jacksonFoltzImg}
+              title = {"Jackson Foltz"}
+              description={"Nonprofit Liaison"}
+              info = {"#jackson"}
+              id = {"jackson"}
+              infoadded={"my name is jackson foltz"}
+              />
+              <ProfileCard
+              imgUrl = {ivyWangImg}
+              title = {"Ivy Wang"}
+              description={"Chief Marketing Officer"}
+              info = {"#ivy"}
+              id = {"ivy"}
+              infoadded={"my name is ivy wang"}
+              />
+              <ProfileCard
+              imgUrl = {jessicaHungImg}
+              title = {"Jessica Hung"}
+              description={"Director of Marketing"} 
+              info = {"#jessica"}
+              id = {"jessica"}
+              infoadded={"my name is jessica Hung"}
+              />
+              <ProfileCard
+              imgUrl = {johnBaekImg}
+              title = {"John Baek"}
+              description={"Fundraiser & Grant Writer"}
+              info = {"#john"}
+              id = {"john"}
+              infoadded={"my name is john baek"}
+              />
+              <ProfileCard
+              imgUrl = {sambaDiaImg}
```

Comment:
Can you also organize the cards by section?
Here are the sections we want:
Leadership and Board
Connor, Steffen, Daniel, Ivy, Michael, Emma
Board: Jackson, Jessica

Engineering
Gagan, Nick, Jason, Sarinna, and Tirtha

Design
Ying and Trang

Product
Cora

Legal and Ops
Samba

Marketing
Sean
Carolyn

Reply:
Where should I put Kevin Chu?